payments-model/src/main/java/com/stripe/android/model/CardBrand.kt

```diff
@@ -157,9 +157,9 @@ enum class CardBrand(
     ),
 
     CartesBancaires(
-        "cartesbancaire",
+        "cartes_bancaires",
```

Comment:
Is there any concern with this change? Basically, I noticed that `cartes_bancaires` is the correct code for this brand, not `cartesbancaire`.